can/parser.cc

```diff
@@ -98,6 +100,13 @@ CANParser::CANParser(int abus, const std::string& dbc_name, const std::vector<st
   bus_timeout_threshold = std::numeric_limits<uint64_t>::max();
 
   for (const auto& [address, frequency] : messages) {
+    // disallow duplicate message checks
+    if (message_states.find(address) != message_states.end()) { 
+      std::stringstream is;
+      is << "Duplicate Message Check: " << address;
+      throw std::runtime_error(is.str());
+    }
+
```

Comment:
might make more sense in the cython wrapper?

Reply:
wouldn't we want to enfore this in C too? I can move it if you think its better in Python

Reply:
this is the right place. the wrapper does too much